plugin-api.d.ts

```diff
@@ -822,26 +829,14 @@ interface Easing {
     | 'EASE_OUT_BACK'
     | 'EASE_IN_AND_OUT_BACK'
     | 'CUSTOM_CUBIC_BEZIER'
-    | 'GENTLE'
-    | 'QUICK'
-    | 'BOUNCY'
-    | 'SLOW'
-    | 'CUSTOM_SPRING'
   readonly easingFunctionCubicBezier?: EasingFunctionBezier
-  readonly easingFunctionSpring?: EasingFunctionSpring
```

Comment:
was this meant to be removed as part of this PR?

Reply:
ah oops, need to rebase